### DISCUSSION TITLE
生成的模型是否与3.0 32k兼容

### DISCUSSION THREAD

**Discussion by mzltest** [svc-develop-team/so-vits-svc#164] [Q&A]
如题，话说最新的版本（就是直接clone过来的）跟3.0版本32k分支是否向下兼容，主要是现在偷的space（https://huggingface.co/spaces/yukie/yukie-sovits3 ）用的3.0 32k分支的模型，不知道直接替换模型会不会导致不兼容。
如果不兼容的话有没有现成脚本把模型转成与3.0 32k兼容的版本，或者有无现成的可以直接作为space部署的脚本

应该不算issue就来discussion问问

**Comment by Miuzarte** [comment]
https://huggingface.co/spaces/Miuzarte/SUI-svc-4.0

最初版的sovits4，缺很多功能

**Reply by mzltest** [reply]
好space，先mark下，哪天偷了去加个YouTube音频搜索和lalal那边的人声分离实现歌姬
